lib/rubocop/target_finder.rb

```diff
@@ -89,9 +89,10 @@ def find_files(base_dir, flags)
       wanted_toplevel_dirs = toplevel_dirs(base_dir, flags) -
                              excluded_dirs(base_dir)
 
-      pattern = if wanted_toplevel_dirs.empty?
-                  # We need this special case to avoid creating the pattern
-                  # /**/* which searches the entire file system.
+      pattern = if wanted_toplevel_dirs.empty? ||
+                   wanted_toplevel_dirs.any? { |dir| dir =~ /,/ }
```

Comment:
The problem in question might be solved by `dir.end_with?(',')`. But having a comma in the middle of a path will surely lead to other problems, when [`Dir.glob` uses the comma for pattern alternation](http://www.ruby-doc.org/core-2.1.5/Dir.html#method-c-glob).